### DISCUSSION TITLE
Week 5 Thoughts/Project

### DISCUSSION THREAD

**Discussion by derekscholes** [USU-ITLS-x265-23-summer/class-repo#112] [General]
# Thoughts
### FreeCodeCamp
- JS
  - I liked that it was very basic and separated functions to a minimum   
- HTML Registration
  -  It was very helpful in understanding input basics. 
### Flexbox Froggy     
- I liked that basic CSS and how you can see each change within the game. 

### FreeCodeCamp
- JS
  - I didn’t like that some lessons were vague and almost impossible without looking up answers   
- HTML Registration
  -  I felt like they made it more complex then it needed to be, and the complexities weren’t even explained. Maybe it will make sense later on. 
### Flexbox Froggy     
- I felt like I didn’t really understand how CSS applied in the real world but as I used some of this in my own web development it started to make more sense.

### FreeCodeCamp
- JS
  - I would love to see how the live changes affect an actual page as it still confuses me how Java causes any change.    
- HTML Registration
  -  I would prefer that they keep it simple and have better explanations to why certain things are added to the page.
### Flexbox Froggy     
-  I don’t know if I would make any changes. It was simple enough.

# Project
## GI/Nutrition Site & Blog w/ CV
- HTML
  - My idea is to have an html page for each distinct portion. I was thinking one page for my CV that includes a download option to download my CV directly plus links to each of my publications and links to references (this will mostly be copying and pasting my CV to the page)
  - My blog I wasn’t sure if I wanted one file that contained all my posts with a button that takes you directly to each post, or one main table of contents page that links you to each post which will all be on separate html files. 
  - For the GI/nutrition website I want to include the pages that are in my mock site and maybe link it to a team page that has my personal statement and another link to my CV. In addition, it will contain the header, footer, and main body portion which will include posts such as the blog or therapy options. 
- CSS 
  - I plan on using CSS to organize pictures, documents, color, and forms throughout my pages. An important CSS point is organizing the header and footer so it is appealing as well as making enough contrast to catch the eye. 
- JavaScript  
  - I honestly don’t know how to include Java in my site still. I was thinking of maybe creating a side menu on my page so that the menu bar can be smaller. This can be for any of the project ideas.

**Comment by kassyoveson** [comment]
Holy Cow Derek. You have already started a mock project site. You are a rock star! Good job.

**Comment by metasean** [comment]
> HTML Registration
>
>    I felt like they made it more complex then it needed to be, and the complexities weren’t even explained. Maybe it will make sense later on.

They do make some aspects of their tutorial projects a bit more complex than necessary, but when they do so, it tends to be laying down the foundation for something that will be coming up soon.


re: projects
It looks like you have a good grasp on the bigger picture.  As for including JavaScript, basically anything interactivity that isn't a link, is likely to use JavaScript.

fwiw: `Java !== JavaScript` (just like `Car !== Carpet`)
